README.md

```diff
@@ -26,7 +26,7 @@ This demo is made with Debian Linux in mind, but the same requirements should ap
 2. Install [Cargo Contract](https://github.com/paritytech/cargo-contract)
 3. Install npm (I recommend using [NVM](https://github.com/nvm-sh/nvm))
 4. Install an Ethereum wallet and a Substrate wallet to your browser (we will use MetaMask and Talisman on Firefox for this example)
-5. Clone the [Astar collator node](https://github.com/AstarNetwork/Astar/tree/feature/pallet-xvm-v2) and check out the `feature/pallet-xvm-v2` branch
+5. Clone the [Astar collator node](https://github.com/AstarNetwork/Astar)
```

Comment:
@hoonsubin I've updated here since I had no issues with the main branch.

Reply:
Perhaps you can change this line later if you want to change it to a particular branch (there is no more `feature/pallet-xvm-v2` branch).